src/components/PresentRankingItem.tsx

```diff
@@ -1,6 +1,9 @@
-import { GOODS_DATA } from '@assets/GOODS_DATA';
-import type { Goods } from '@assets/GOODS_DATA';
+import { GOODS_DATA } from '@/assets/goodsData';
+import type { Goods } from '@/assets/goodsData';
 import styled from '@emotion/styled';
+const basicRankingComponentNumber = 6;
+const ManyRankingComponentNumber = 18;
```

Comment:
요부분은 고정된 상수값이니까 상수를 명확히 드러내기 위해서 대문자 스네이크 케이스로 작성하는게 좋아용!
해당 컴포넌트 말고도 다른 컴포넌트에서도 사용될 것 같으면 아예 파일 위치도 같이 분리하는 것도 좋구용!

```suggestion
const BASIC_RANKING_COMPONENT_NUMBER = 6;
const MANY_RANKING_COMPONENT_NUMBER = 18;
```